fbpic/boundaries/boundary_communicator.py

```diff
@@ -548,7 +551,7 @@ def exchange_fields( self, interp, fieldtype, method ):
             A list of InterpolationGrid objects
             (one element per azimuthal mode)
 
-        fieldtype: str
+        fldtype: str
```

Comment:
I shortened the name to make the code more readable / have shorter lines of code.